src/translations/en.json

```diff
@@ -1,7 +1,7 @@
 {
   "app.components.Header.DevicesLabel": "Devices",
   "app.components.Header.LanguageEnglishLabel": "English",
-  "app.components.Header.LanguageSpanishLabel": "Spaninsh",
```

Comment:
@bonavida nominee to Disflexia Evangelist II.

Reply:
😢